sky/engine/core/dart/mojo_services.dart

```diff
@@ -4,8 +4,24 @@
 
 part of dart_ui;
 
-int takeRootBundleHandle() native "takeRootBundleHandle";
-int takeServicesProvidedByEmbedder() native "takeServicesProvidedByEmbedder";
-int takeServicesProvidedToEmbedder() native "takeServicesProvidedToEmbedder";
-int takeShellProxyHandle() native "takeShellProxyHandle";
-int takeViewHandle() native "takeViewHandle";
+/// Mojo handles provided to the application at startup.
+///
+/// The application can take ownership of these handles by calling the static
+/// "take" functions on this object. Once taken, the application is responsible
+/// for managing the handles.
+class MojoServices {
+  MojoServices._();
+
+  static int takeRootBundle() native "MojoServices_takeRootBundle";
```

Comment:
would be helpful to have dartdocs for each one here saying wtf they are. I tried to write the docs the other day but couldn't really figure it out.